examples/2d/wireframe_2d.rs

```diff
@@ -66,16 +69,25 @@ fn setup(
         NoWireframe2d,
     ));
     // Rectangle: Follows global wireframe setting
+    let rectangle_mesh = meshes.add(Rectangle::new(100.0, 100.0));
     commands.spawn((
-        Mesh2d(meshes.add(Rectangle::new(100.0, 100.0))),
+        Mesh2d(rectangle_mesh.clone()),
         MeshMaterial2d(materials.add(Color::BLACK)),
         Transform::from_xyz(0.0, 0.0, 0.0),
+        NoWireframe2d,
     ));
+    commands.spawn((Mesh2d(rectangle_mesh), Transform::from_xyz(0.0, 0.0, 0.0)));
```

Comment:
I think it would be slightly clearer if these were spawned as children, then if this code is copy pasted the wireframe will move along with the original.

Reply:
Also Transform is a required component for Mesh2d so it doesn't need to be added (and if it did you could use Transform::default() instead :P)